spec/lib/definition/types/lambda_spec.rb

```diff
@@ -17,6 +17,40 @@
     end
   end
 
+  describe "when definition defines a custom error message" do
+    let(:test_lambda) do
+      lambda do |value|
+        if value.even?
+          conform_with(value)
+        else
+          fail_with("Value is not even")
+        end
+      end
+    end
+
+    describe ".conform" do
+      subject(:conform) { definition.conform(value) }
+
+      context "with even value" do
+        let(:value) { 2 }
+
+        it "conforms" do
+          expect(conform).to conform_with(value)
+        end
+      end
+
+      context "with odd value" do
+        let(:value) { 1 }
+
+        it "does not conform" do
```

Comment:
Maybe to make this part a bit more clear
```suggestion
        it "does not conform using custom error message" do
```